tests/sentry/integrations/api/endpoints/test_organization_integrations.py

```diff
@@ -51,3 +69,20 @@ def test_provider_key(self):
             self.organization.slug, qs_params={"provider_key": "vercel"}
         )
         assert response.data == []
+
+    def test_integration_type(self):
```

Comment:
nit: maybe add a test for what happens when u pass both `providerKey` and `integrationType`
also maybe throw docs in ur backlog

Reply:
what is the preferred behavior when both `providerKey` and `integrationType` are passed in?